project_issue_sheet_extended/README.rst

```diff
@@ -0,0 +1,42 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Project issue sheet extended
+============================
+
+Add ability to timesheet on issue in the detail tab of the timesheet
+il also add the possibility to timesheet on a child analytic account of
+an account linked with a project.
+
+
```

Comment:
Add Installation, configuration and usage section as described here https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst